content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -488,14 +478,15 @@ The folder structure for a _$JENKINS_HOME_ directory is as follows:
 JENKINS_HOME
  +- config.xml     (Jenkins root configuration file)
  +- *.xml          (other site-wide configuration files)
- +- identity.key   (RSA key pair that identifies an instance)
```

Comment:
I think this should be retained as `identity.key.enc` since the file has been renamed